admin/controllers/administrator.js

```diff
@@ -9,7 +9,7 @@ const checkWindowErrorMessages = require('../lib/errors/check-window')
 const checkWindowService = require('../services/check-window.service')
 const dateService = require('../services/date.service')
 const checkWindowDataService = require('../services/data-access/check-window.data.service')
-const sortingUiService = require('../services/sorting-ui.service')
+const sortingUIService = require('../services/sorting-ui.service')
```

Comment:
@GeorgeChatzigiannis I believe the correct camel case is the as it was before 'sortingUiService' - I know it's not intuitive but that's the correct syntax. 

From Wikipedia:
"Initialisms of three or more letters are CamelCase instead of upper case (e.g., parseDbmXmlFromIPAddress instead of parseDBMXMLFromIPAddress")
https://en.wikipedia.org/wiki/Naming_convention_(programming)#Challenges

Reply:
i would say sortingUiService is more readable.  can we just approve the damn PR already 😄

Reply:
changed it to sortingAttributesService